src/FlowScene.cpp

```diff
@@ -561,20 +576,12 @@ saveToMemory(QJsonDocument::JsonFormat format) const
 
   sceneJson["connections"] = connectionJsonArray;
 
-  QJsonDocument document(sceneJson);
-
-  return document.toJson(format);
+  return sceneJson;
 }
 
 void
 FlowScene::
-loadFromMemory(const QByteArray& data)
-{
-  const QJsonObject jsonDocument = QJsonDocument::fromJson(data).object();
-  loadFromMemory(jsonDocument);
-}
-
-void FlowScene::loadFromMemory(const QJsonObject& data)
+loadFromObject(const QJsonObject& data)
```

Comment:
loadFromObject(const QJsonObject& data) is redundant because already exists loadFromMemory(const QJsonObject& data). So I should leave loadFromMemory() as it is and remove this method.

Reply:
I considered `loadFromMemory` confusing because it implies `QByteArray`. Renaming it to `loadFromObject` makes it more symmetrical with `saveToObject`, which cannot be renamed to `saveToMemory`. I left `loadFromMemory` for backwards compatibility.

Reply:
`saveToMemory` and `loadFromMemory` cannot be renamed otherwise it would break compatibility with the original library. I understand your point. Could you at least add a brief comment in the new method?

Reply:
I wasn't suggesting to rename `saveToMemory` or `loadFromMemory`, I was explaining the rational behind the names `saveToObject` and `loadFromObject`. I'll add more comments to the functions.

Reply:
Ok when it's done I'm going to merge